docs/core/deploying/index.md

```diff
@@ -14,21 +14,19 @@ ms.assetid: da7a31a0-8072-4f23-82aa-8a19184cb701
 
 # .NET Core Application Deployment #
 
-You can deploy your .NET Core app in either of two ways:
+You can create two types of deployments for .NET Core applications: 
 
-- As a portable app. A portable app relies on a shared system-wide version of .NET Core to be present on the target system. Because .NET Core is already present, your app is portable between installations of .NET Core. Your app contains only its own code and any third-party dependencies that are outside of the .NET Core libraries. Portable applications are .dll files that can be launched by using the [dotnet utility](../tools/dotnet.md) from the command line. For example, `dotnet app.dll` runs a portable application named `app`.
+- Framework-dependent deployment. As the name implies, framework-dependent deployment (FDD for short) relies on a shared system-wide version of .NET Core to be present on the target system. Because .NET Core is already present, your app is also portable between installations of .NET Core. Your app contains only its own code and any third-party dependencies that are outside of the .NET Core libraries. FDDs contain .dll files that can be launched by using the [dotnet utility](../tools/dotnet.md) from the command line. For example, `dotnet app.dll` runs an application named `app`.
```

Comment:
acronyms are introduced without the need to say for short:
framework-dependent deployment (FDD for short) -> framework-dependent deployment (FDD)